### ISSUE TITLE
关于运行代码的问题

### ISSUE THREAD

**Issue by luckyzhangqian** [chunyuanY/RumorDetection#15]
is:issue is:open 新人小白，刚刚入门谣言检测，环境已经配好了，但运行run.py报[Errno 2] No such file or directory: 'checkpoint/weights.best.weibo.glan'这个错，请问这个错误遇到过吗？不应该是自己生成的吗？谢谢

**Comment by bazarijoo** [comment]
Yes, uncomment [this line](https://github.com/chunyuanY/RumorDetection/blob/d882f30190a48adcc9e576df1d85be162deb60bc/run.py#L29) and you will be fine.

**Comment by XXXJH01** [comment]
您好，为什么运行结束后，没有自动生成weights.best.weibo.glan文件呢
